numpy/random/mtrand.pyx

```diff
@@ -763,7 +763,7 @@ cdef class RandomState:
         else:
             raise TypeError('Unsupported dtype %r for randint' % _dtype)
 
-        if size is None and dtype in (bool, int, np.compat.long):
+        if size is None and dtype in (bool, int):
```

Comment:
Is `np.compat.long` always `int` even on platforms like windows?

Reply:
yes
https://github.com/numpy/numpy/blob/47df8f84d48357bc1d6b0d353c4aca5f7bc55428/numpy/compat/py3k.py#L28